extensions/amp-sidebar/1.0/test/test-component.js

```diff
@@ -741,63 +720,19 @@ describes.sandboxed('Sidebar preact component', {}, (env) => {
               </ul>
             </SidebarToolbar>
           </Sidebar>
-          <button id="open" onClick={() => ref.current.open()}></button>
-          <button id="close" onClick={() => ref.current.close()}></button>
         </>
       );
 
-      const openButton = wrapper.find('#open');
-      const closeButton = wrapper.find('#close');
-
-      // verify sidebar is closed
-      let sidebarElement = wrapper.find(Sidebar).getDOMNode();
-      expect(isOpened(sidebarElement)).to.be.false;
-
-      // Toolbar target should have children
+      // Toolbar target now has 3 nodes, 1 existing, 2 appended from toolbar
       expect(target.hasChildNodes()).to.be.true;
-
-      // click to open the sidebar
-      openButton.simulate('click');
-
-      // verify sidebar is opened and toolbar has children
-      sidebarElement = wrapper.find(Sidebar).getDOMNode();
-      expect(isOpened(sidebarElement)).to.be.true;
-      expect(target.hasChildNodes()).to.be.true;
-
-      // click to close the sidebar
-      closeButton.simulate('click');
-
-      // verify sidebar is closed and toolbar has children
-      sidebarElement = wrapper.find(Sidebar).getDOMNode();
-      expect(isOpened(sidebarElement)).to.be.false;
-      expect(target.hasChildNodes()).to.be.true;
-
-      Element.prototype.animate = animateFunction;
+      expect(target.childElementCount).to.equal(3);
+      expect(target.children[1].nodeName).to.equal('NAV');
+      expect(target.lastElementChild.nodeName).to.equal('STYLE');
     });
 
-    it('toolbar target content updates when media query truthiness change', () => {
-      // disable animations for synchronous testing
-      const animateFunction = Element.prototype.animate;
-      Element.prototype.animate = null;
-
-      // mock the media query to manually update matching
-      let mockMediaQueryList;
-      window.matchMedia = (media) => {
-        const mql = matchMediaFunction(media);
-        mockMediaQueryList = {
-          matches: mql.matches,
-          addEventListener: function (type, callback) {
-            this.callback = callback;
-          },
-          removeEventListener: function () {
-            this.callback = null;
-          },
-        };
-        return mockMediaQueryList;
-      };
-
-      // this media query is always true
-      mediaQuery = '';
+    it('verify toolbar target content is removed on unmount', () => {
+      // this media query is always false
+      mediaQuery = 'false';
```

Comment:
suggestion: Let's make this a valid media query that is always false, i.e. `(max-height: 0px)` instead of the value `false`?

Reply:
Updated!